examples/msbr/openmc_msbr_model.py

```diff
@@ -458,10 +465,11 @@ def plot_geometry(name,
                         'range': (800, 1000)}
 
 ll, ur = geo.root_universe.bounding_box
-msbr_volume_calc = openmc.VolumeCalculation([fuel, moder], 1000000000, ll, ur)
-#msbr_volume_calc.set_trigger(1e-03, 'rel_err')
-settings.volume_calculations = [msbr_volume_calc]
-settings.run_mode = 'volume'
+if volume:
+    msbr_volume_calc = openmc.VolumeCalculation([fuel, moder], 1000000000, ll, ur)
```

Comment:
I included the `int` because I'm not sure if it's required for this parameter. But I think this format is clearer. Counting zeros might be error prone.
```suggestion
    msbr_volume_calc = openmc.VolumeCalculation([fuel, moder], int(1e10), ll, ur)
```

Reply:
Great suggestion